src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import TabsItem from './TabsItem';
+import { StyledTabs } from './styles/Tabs';
+
+const Tabs = props => {
+    const tabsValue = props.children[1].props.children.props.children;
+    const [description, setDescription] = useState(tabsValue);
+
+    console.log('%cTabs tabsValue: ', 'color:blue', tabsValue);
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        const title = e.target.innerHTML;
```

Comment:
To rozwiązanie jest bliższe czystemu JSowi niż Reactowi. Lepiej opierać się o propsy i dane przekazywane w parametrach. Można też użyć referencji.
Pamiętaj, że możesz korzystać z `useContext`, który pozwoli Ci przekazywać do dzieci odpowiednie dane/metody.